llama_stack/apis/eval/eval.py

```diff
@@ -61,15 +61,42 @@ class EvaluateResponse(BaseModel):
     scores: Dict[str, ScoringResult]
 
 
+@json_schema_type
+class EvalJob(CommonJobFields):
+    """
+    An evaluation job.
+
+    :param type: The type of the job.
+    :param result_files: The file ids of the eval results.
+    :param result_datasets: The ids of the datasets containing the eval results.
+    :param benchmark_id: The id of the benchmark to evaluate on.
+    :param candidate: The candidate to evaluate on.
+    """
+
+    type: JobType = JobType.evaluation.value
+    result_files: List[str] = Field(
```

Comment:
I wonder if this field should be "common". Are there any jobs where it wouldn't make sense? (Or at least where an empty results list wouldn't be satisfactory?)

Reply:
Another question (perhaps for future discussions): what happens to these files / datasets referred if the user deletes / modifies them? Will job still refer to them? How can the user of the API confirm that these files / datasets still represent the result of the job execution at the moment of the execution? (I'm wondering if we will need a validation mechanism like checksum to at least give the user a hint.)

Reply:
Thanks! I've just update it to artifacts according to your PR

Reply:
what are result_files for an inference batch?

Reply:
(spitballing) a reference to /file that contains the answers?